build.yaml

```diff
@@ -235,7 +241,9 @@ common_cli:
     # during the bootstrapping of the slave.
     # This file is located in /etc/jenkins_slave on the Master box.
     # These variables contain the PIP URL and IP address of our caching server.
-    . /tmp/pip.sh
+    if [ -e /tmp/pip.sh ]; then
+      . /tmp/pip.sh
+    fi
 
```

Comment:
only set the PIP caching if the file exists (currently not the case on Mesos and OSX)